MVCDemos/Models/SampleData/ContextMenuData.cs

```diff
@@ -1,1243 +1,358 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
 
 namespace DevExtreme.MVC.Demos.Models.SampleData {
     public static class ContextMenuData {
         public static readonly IEnumerable<ContextMenuScrollingItem> Products = new[] {
             new ContextMenuScrollingItem {
-                text = "Audio",
+                text = "Electronics",
```

Comment:
The dataSource for both demos is identical. We might be able to use only one file, but let's leave it as it is. I don't see any plans to refactor this.

Reply:
You can resolve my comment